server/src/main/scala/controllers/Application.scala

```diff
@@ -501,7 +501,9 @@ class Application @Inject()(implicit val config: Configuration,
 
     log.info(s"Start hour: $startHour End hour: $endHour")
 
-    val fileName = f"$terminalName-desks-and-queues-${pit.getFullYear()}-${pit.getMonth()}%02d-${pit.getDate()}%02dT" +
+    val portCode = config.getString("portcode").getOrElse("PORT-CODE-MISSING").toUpperCase
```

Comment:
we have the port code in airportConfig.portCode so don't need any of these config lookups